DNN Platform/DotNetNuke.Web/Api/Auth/AuthMessageHandlerBase.cs

```diff
@@ -91,24 +183,24 @@ protected static bool IsXmlHttpRequest(HttpRequestMessage request)
             return !string.IsNullOrEmpty(value) &&
                    value.Equals("XmlHttpRequest", StringComparison.InvariantCultureIgnoreCase);
         }
-
+
         /// <summary>
-        /// Sets the current principal for the request.
+        /// Sets the current principal.
         /// </summary>
-        /// <param name="principal">The principal to set.</param>
-        /// <param name="request">The current request.</param>
+        /// <param name="principal">The pricipal to set.</param>
+        /// <param name="request">The http request message.</param>
         protected static void SetCurrentPrincipal(IPrincipal principal, HttpRequestMessage request)
         {
             Thread.CurrentPrincipal = principal;
             request.GetHttpContext().User = principal;
         }
-
+
         /// <summary>
-        /// Asynchronously sends a response.
+        /// Sends an HTTP response asynchronously.

```

Comment:
```suggestion
        /// Sends an HTTP request asynchronously.
```

Reply:
This returns an HttpResponseMessage no ?

Reply:
It sends a request (from the parameters) and returns a response (right?)

Reply:
Hmm, I'd say it handles a request by sending a response for it... Or am I confused...

Reply:
This overrides and ultimately delegates to [`HttpMessageHandler.SendAsync`](https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpmessagehandler.sendasync), and that method's docs say
>Send an HTTP request as an asynchronous operation.

Reply:
Ok, makes sense, thanks.